scripts/install-vim

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+# Install and setup a Vim or Neovim for running tests.
+# This should work on both Travis and people's desktop computers, and be 100%
+# independent from any system installed Vim.
+#
+# It will echo the full path to a Vim binary, e.g.:
+#   /some/path/src/vim
+
+set -euC
+cd "$(dirname "$0")"
+
+vimgodir="$(dirname "$(readlink -f .)")" # TODO: doesn't work on OSX I think.
```

Comment:
Can you add an if clause that exits the script when run on OSX (checking with uname for example). I'm using macOS and don't want to blow my setup because of this. We can later add homebrew or a different kind of installation step here.

Reply:
I just need to fix it; `readlink -f` doesn't work on OSX but there's a workaround (I just couldn't remember what it was exactly yesterday).

Reply:
Is `$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )` what you're thinking of @Carpetsmoker ?